tests/end-to-end/test-language-detect.bats

```diff
@@ -0,0 +1,65 @@
+setup() {
+  load "${BATS_LIBS_ROOT}/bats-support/load"
+  load "${BATS_LIBS_ROOT}/bats-assert/load"
+}
+
+@test "can detect C# language" {
+    #cd tests/end-to-end/test-data/Csharp_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] The following language(s) support secrets detection: C#'
+    assert_output --partial '[seCureLI] - C#: 100%'
+}
+
+@test "can detect Go language" {
+    #cd tests/end-to-end/test-data/Go_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - Go: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): Go.'
+}
+
+@test "can detect Javascript language" {
+    #cd tests/end-to-end/test-data/JavaScript_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - JavaScript: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): JavaScript.'
+}
+
+@test "can detect Kotlin language" {
+    #cd tests/end-to-end/test-data/Kotlin_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - Kotlin: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): Kotlin.'
+}
+
+@test "can detect Python language" {
+    #cd tests/end-to-end/test-data/Python_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - Python: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): Python.'
+}
+
+@test "can detect Swift language" {
+    #cd tests/end-to-end/test-data/Swift_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): Swift.'
+    assert_output --partial '[seCureLI] - Swift: 100%'
+}
+
+@test "can detect Terraform language" {
+    #cd tests/end-to-end/test-data/Terraform_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - Terraform: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): Terraform.'
+}
+
+@test "can detect Typescript language" {
+    #cd tests/end-to-end/test-data/Typescript_Sample/
+    run python secureli/main.py init -ry
+    assert_output --partial '[seCureLI] - TypeScript: 100%'
+    assert_output --partial '[seCureLI] seCureLI has been installed successfully for the following language(s): TypeScript.cd ..'
+}
+
+@test "can detect Cloudformation language" {
```

Comment:
I think you can remove this test per our discussion last week. 
https://slalom.slack.com/archives/C03M8K809AM/p1712954689296749